Readme.md

```diff
@@ -27,9 +25,10 @@ g.load('/path/to/your/thesaurus')
 ```
 First, define the type URI for descriptors.
 If your thesaurus has a hierarchical structure that includes groups, 
```

Comment:
Suggestion (?): "If your thesaurus is structured into sub-thesauri by providing categories for the concepts of the thesaurus using, e.g., `skos:Collection`, you can optionally specify the type of these categories via a URI.
In this case you should also specify the relation that relates concepts to categories."